packages/lesswrong/lib/collections/messages/newSchema.ts

```diff
@@ -97,6 +100,51 @@ const schema = {
       },
     },
   },
+  currentUserVote: DEFAULT_CURRENT_USER_VOTE_FIELD,
+  currentUserExtendedVote: DEFAULT_CURRENT_USER_EXTENDED_VOTE_FIELD,
+  allVotes: {
+    graphql: {
+      outputType: "[Vote!]",
+      canRead: ["guests"],
+      resolver: async (document, args, context) => {
+        const { currentUser } = context;
+        if (userIsAdminOrMod(currentUser)) {
+          return await getAllVotes(document, context);
+        } else {
+          return await getCurrentUserVotes(document, context);
+        }
+      },
+    },
+  },
+  voteCount: defaultVoteCountField('Messages'),
+  baseScore: {
```

Comment:
`baseScore` should just be `DEFAULT_BASE_SCORE_FIELD` (along with all of the rest)